app/components/preprint-form-authors.js

```diff
@@ -129,39 +94,15 @@ export default CpPanelBodyComponent.extend(Analytics, {
                     label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Add Author`,
                 });
             this.set('user', user);
-            this.attrs.addContributor(user.id, 'write', true, this.get('sendEmail'), undefined, undefined, true)
+            this.get('model').addContributor(user.id, 'write', true, this.get('sendEmail'), undefined, undefined, true)
```

Comment:
The submit controller was previously using the NodeMixin from `ember-osf` which had contributor methods on them that called functions on the node model.  Rather than making a PreprintMixin or similar, I have added contributor methods to the preprint model in `ember-osf` and calling them directly here.